src/game-support/trackmania.md

```diff
@@ -4,9 +4,9 @@
 {{#template ../templates/rating.md status=Silver installs=Yes opens=Yes}}
```

Comment:
If "practically nothing is rendered properly" , I would suggest changing the status to Bronze, as that sounds to me like it "severely limits" playability.

Reply:
@YiraSan I agree with @hahayupgit. Can you change it and clarify the `NOTE` section below?